demos/nodejs/hapi/config.js.sample

```diff
@@ -0,0 +1,8 @@
+const config = {
+  apiDomain: 'api.loginradius.com',
+  apiKey: '<API_KEY>',
+  apiSecret: '<API_SECRET'>,
+  siteName: <APP_NAME'>',

```

Comment:
@karmanya007 
Please correct the misplaced single quotes in this and express folder.